packages/ui/core-components/src/lib/atoms/inputs/dropdown/Dropdown.svelte

```diff
@@ -121,6 +122,9 @@
 
 	$: hasHadSelection = hasHadSelection || $selectedOptions.length > 0;
 	$: if ($selectedOptions && hasHadSelection) {
+		if ($selectedOptions.length > opts.length) {
```

Comment:
Is this condition right? I played around with it a bit and it seems to be working correctly, but the condition just seems a little off. Would it make more sense to check if there is a selected option that no longer exists in `opts`?

```js
const selectedOptionsNoLongerInOpts = $selectedOptions.filter(
    (x) => !opts.some((y) => y.value === x.value)
);
```